visualization/setupJest.ts

```diff
@@ -9,3 +10,8 @@ if (process.env.TEST_TYPE !== "e2e") {
 	require("angular-mocks")
 	require("jest-preset-angular/setup-jest")
 }
+
+const numberToLocaleString = Number.prototype.toLocaleString
+Number.prototype.toLocaleString = function (locale = "en-US") {
+	return numberToLocaleString.call(this, locale)
+}
```

Comment:
setupJest.ts is loaded by our test framework jest. Without it `(9001).toLocaleString()` produces, depended on the set language of the executing system, `"9,001"` or `"9.001`. E.g. new added test for checking formating of color range succeded on my machine when tested against `"9,001"` but failed in the pipeline. This sets a fixed language for every `(number).toLocaleString`, so that the test will pass on any pc regardless its language.

P.S. 9001 is over 9000 :D